ebusd-2.1.x/de/vaillant/06.vms.csv

```diff
@@ -0,0 +1,41 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
+#,VMS01,VMS01,0020071488 243,,,,,,,,,,
```

Comment:
please check the product code and enter it here instead of 0020071488. this is revealed by `ebusctl f -c scan.06 -v id` as 'product=...'

Reply:
Got this answer:
`scan.06 id = prefix=??;year=??;week=??;product=??????????;supplier=????;counter=??????;suffix=??`